docs/source/commands.rst

```diff
@@ -0,0 +1,86 @@
+Remote Scan Commands
+~~~~~~~~~~~~~~~~~~~~~
+This file documents which programs on the remote host are used to collect different groups of facts.  In addition to the programs below, we depend on standard shell facilities like those provided by bash.
```

Comment:
I feel like we shouldn't say `This file` since it is a section. Calling the commands programs also reads weird to me but that is not a blocker